_articles/getting_started_clients.md

```diff
@@ -48,7 +48,7 @@ For complete documentation of the Python client, visit the [`synapseclient` docs
 The `synapser` package is available for R version 3.4 and 3.5.
```

Comment:
`synapser` should be available for R version 4.0. @jkiang13 should we update the language here?